packages/node/src/plugins/LoadFileChunkLoadingRuntimeModule.ts

```diff
@@ -272,28 +274,29 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
                         'var promise = new Promise(async function(resolve, reject) {',
                         Template.indent([
                           'installedChunkData = installedChunks[chunkId] = [resolve, reject];',
-                          `var filename = typeof process !== "undefined" ? require('path').join(__dirname, ${JSON.stringify(
```

Comment:
Because this plugin is in the node environment, so we can no need to judge the process if exist ?